cli/commands/patchMigrations.js

```diff
@@ -0,0 +1,73 @@
+var Promise = require('bluebird');
+
+var fs = Promise.promisifyAll(require('fs'));
+
+let frameworkName = 'Transmute Framework'
+
+let patchBegin = `🦄 ${frameworkName}`
+let patchEnd = `🐩 ${frameworkName}`
+
+function replaceAll(str, find, replace) {
+  return str.replace(new RegExp(find, 'g'), replace);
+}
+
+const backupPatchTarget = (targetPath) =>{
+    let contents
```

Comment:
you will be unable to read future prs, they will sound like dungeons and dragons had sex with json.